### ISSUE TITLE
Breaks sidebar navigation in Sublime Text 4

### ISSUE THREAD

**Issue by mjrodgers** [bordaigorl/sublime-non-text-files#8]
I'm new to using Sublime Text/this package so I don't know if this is pre-existing behavior or something related to Sublime Text 3-4 update, but with this package installed it is completely impossible to use sidebar navigation in Sublime Text 4. (I'm on macOS.)

- Press Ctrl-0 to give focus to sidebar
- Press up/down to get to a non-text file

What should happen: Should be able navigate sidebar/File Tree with up/down arrows 

What happens instead: When I get to a non-text file, it can't even be highlighted so the navigation kicks back to the previous text file and focus instantly reverts back to the editor window.

**Comment by bordaigorl** [comment]
I agree this is undesirable, but I don't know how to fix it. Suggestions/PRs welcome.
